app/code/community/Bolt/Boltpay/Helper/Bugsnag.php

```diff
@@ -164,4 +164,15 @@ protected function getBoltTraceId()
         return null;
     }
 
+   /* add metaData for to create new tab
+    *
+    * @param array $metaData
+    * @param Boolean $merge
+    *
+    */
+    public function addMetaData(array $metaData, $merge = false)
```

Comment:
@chirag-taraviya what does `$merge` param do? Can we remove this and hard-code true at line 175?